.ci.yaml

```diff
@@ -6,6 +6,7 @@
 # More information at:
 #  * https://github.com/flutter/cocoon/blob/main/CI_YAML.md
 enabled_branches:
+  - flutter-2.8-candidate.16
```

Comment:
This shouldn't be necessary as it matches  ` - flutter-\d+\.\d+-candidate\.\d+`

Reply:
Fixed